src/OrchardCore.Modules/OrchardCore.Rewrite/Startup.cs

```diff
@@ -0,0 +1,60 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Rewrite;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using OrchardCore.Admin;
+using OrchardCore.Navigation;
+using OrchardCore.Security.Permissions;
+using OrchardCore.Settings;
+using OrchardCore.Rewrite.Rules;
+using OrchardCore.Modules;
+using OrchardCore.Rewrite.Models;
+using OrchardCore.Rewrite.Drivers;
+using OrchardCore.DisplayManagement.Handlers;
+
+namespace OrchardCore.Rewrite;
+
+public sealed class Startup : StartupBase
+{
+    public override int Order
```

Comment:
If this important, please document why it is important, otherwise, remove the order.

Reply:
This module must have precedence before things like "Media" (and probably other) so even alternative media url can be provided. For example in my use case I have legacy website and would like to rewrite all urls: /img/... to /media/img/....  Probably this module should have it's own Order constant like some other modules have. What do you think?